OpenStack-Rabbit-Consumer/usr/local/bin/message_consumer.py

```diff
@@ -17,17 +17,43 @@ def is_aq_message(message):
     is for an Aquilon VM.
     """
     metadata = message.get("payload").get("metadata")
+    print(metadata)
     if metadata:
         if set(metadata.keys()).intersection(['AQ_DOMAIN', 'AQ_SANDBOX', 'AQ_OSVERSION', 'AQ_PERSONALITY', 'AQ_ARCHETYPE', 'AQ_OS']):
             return True
+    if metadata:
+        if set(metadata.keys()).intersection(['aq_domain', 'aq_sandbox', 'aq_osversion', 'aq_personality', 'aq_archetype', 'aq_os']):
```

Comment:
You can wrap stuff in brackets over multiple lines. This goes off the edge!

Reply:
Why not store `['aq_domain', 'aq_sandbox', 'aq_osversion', 'aq_personality', 'aq_archetype', 'aq_os']` in a constant and then re-use it for these bits.

You could also use list comprehension for the upper case version:
```python
AQ_OBJECTS = [
    'aq_domain',
    'aq_sandbox',
    'aq_osversion',
    'aq_personality',
    'aq_archetype',
    'aq_os',
]

…

if set(metadata.keys()).intersection([s.lower for s in AQ_OBJECTS]):
…
```

Reply:
That would also work. :)